Cargo.toml

```diff
@@ -47,6 +47,11 @@ strict_encoding_derive = "1"
 lightning_encoding = "0.4.0-beta.1"
 client_side_validation = "0.4"
 descriptor-wallet = "0.4"
+#    We use these to simplify enum strict encoding, but once the required
+#    functionality will become part of either amplify_derive or lnpbp_derive
+#    crates they should be removed
+num-traits = "~0.2.11"
+num-derive = "~0.3.0"
```

Comment:
We should not increase the number of non-important upstream dependencies. I have originally used num-derive crates and it took a while for me to remove the need in them, so NACK to introducing them back

```suggestion
```